search/search-client-query-transformer/src/main/java/io/camunda/search/clients/query/SearchQueryBuilders.java

```diff
@@ -475,7 +476,9 @@ public static <C extends List<Operation<Object>>> List<SearchQuery> variableOper
                   case LOWER_THAN_EQUALS ->
                       range(q -> q.field(varValue).lte(op.value())).toSearchQuery();
                   case IN -> objectTerms(varValue, op.values());
-                  default -> throw unexpectedOperation("Variable", op);
+                  default ->
+                      throw new IllegalStateException(
+                          "Unexpected %s operation: %s".formatted("Variable", op.operator()));
```

Comment:
🔧 Method does the same, let's keep that.

Reply:
Nope, the method can't deal with the new UntypedOperation which we have here.

Reply:
But I just saw, that just the operator is used there. I reused the method there, and changed the method to just need the operator instead the whole operation.